src/devices/Adxl345/README.md

```diff
@@ -0,0 +1,38 @@
+# ADXL345
+ADXL345 is a small, thin, low power, 3-axis accelerometer with high resolution (13-bit) measurement at up to ±16g.
+
+## Sensor Image
+![](sensor.jpg)
+
+## Usage
+```C#
+// SPI bus 0
+// CS Pin connect to CS0(Pin24)
+// set gravity measurement range ±4G
+using (Adxl345 sensor = new Adxl345(0, 0, GravityRange.Range2))
+{
+    // read acceleration
+    Vector3 data = sensor.Acceleration;
+
+    //TODO
```

Comment:
NIT: Instead of `TODO` I would right in here something like `// Use sensor` or something like that. Having TODO kind of means that this README is incomplete